centos/7/Dockerfile

```diff
@@ -22,6 +22,5 @@ EXPOSE 29418 8080
 VOLUME ["/var/gerrit/git", "/var/gerrit/index", "/var/gerrit/cache", "/var/gerrit/db", "/var/gerrit/etc"]
 
 # Start Gerrit
-CMD git config -f /var/gerrit/etc/gerrit.config gerrit.canonicalWebUrl http://$HOSTNAME:8080/ && \
```

Comment:
This is wrong: if you do not define a canonicalWebUrl, it will keep the wrong one anyway, using the container id which is wrong.

Closing this PR anyway because contributions need to go through https://gerrit-review.googlesource.com